src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -114,12 +115,49 @@ class ConsignmentRouteSpec extends AnyFlatSpec with Matchers with TestRequest wi
     checkConsignmentExists(response.data.get.addConsignment.consignmentid.get)
   }
 
+  "addConsignment" should "create a consignment of type 'standard' if no consignment type provided" in {
+    addSeries(
+      UUID.fromString("6e3b76c4-1745-4467-8ac5-b4dd736e1b3e"),
+      bodyId = UUID.fromString("830f0315-e683-440e-90d0-5f4aa60388c6"),
```

Comment:
Is it worth moving the two `UUID.fromString` calls into a variable at the top of the class and referencing it here as this is called quite a few  times?

Reply:
Just moved the two UUID creation calls into variables. It gets too confusing to do much else. It's better to see the workings clearly of each individual test, rather than abstract the tests too much, I think.